test/testutils/kubecdl/kubecdl.go

```diff
@@ -14,68 +17,94 @@ import (
 	"strings"
 	"time"
 
-	. "github.com/onsi/ginkgo"
-	"github.com/solo-io/squash/pkg/models"
-
 	v1 "k8s.io/api/core/v1"
 )
 
 func init() {
 	rand.Seed(time.Now().UTC().UnixNano())
 }
 
-type Kubectl struct {
+type Kubecdl struct {
```

Comment:
cdl?

Reply:
yeah, just being silly :) happy to revert